_authors/prakhar.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Prakhar Gupta
+short_name: PG
+twitter: <twitter handle without the @>
```

Comment:
Can you remove the twitter line and can you move your bio from the `_authors` to the `_authors/osci_2023` directory? Thank you!

Reply:
Should be fixed now